### DISCUSSION TITLE
The required flags `--secret` and `--url` were not specified

### DISCUSSION THREAD

**Discussion by camachom** [umputun/remark42#1664] [Q&A] [answered]
Hello! Thanks for building this awesome comment engine.

I'm running into an issue with a fresh install. After running `docker compose up` as root, I get the following:
```
remark42  | changing ownership of /srv/var to app:app (remark42 user inside the container)
remark42  | execute /srv/remark42 server
remark42  | remark42 v1.12.1-1f2500f-20230821T11:59:46
remark42  | the required flags `--secret' and `--url' were not specified
```

I'm assuming the provided `docker-compose.yml` should be enough to run this container. 

Any ideas? I apologize if I missed something in the instructions or if the fix is obvious, but I've spent a good amount of time and can't figure it out.

**Comment by umputun** [comment, marked as answer]
the provided https://github.com/umputun/remark42/blob/master/docker-compose.yml#L24C1-L26C15 expects user to set those vars in the environment 

_pls use discussion for similar questions. issue is for the real problem what should be fixed, I'll move it_

**Reply by camachom** [reply]
Thank you!
